index.js

```diff
@@ -104,9 +104,16 @@
             throw new Error("Component lacks a handleClickOutside(event) function for processing outside click events.");
           }
 
+          // react@15.x will emit warnings when methods are bound using the
+          // .bind() syntax; this provides a way to delegate to
+          // `handleClickOutside` without React emitting a `console.warn`.
+          var delegateHandler = function() {
+            instance.handleClickOutside.apply(instance, arguments);
+          }
+
           var fn = this.__outsideClickHandler = generateOutsideCheck(
             ReactDOM.findDOMNode(instance),
-            instance.handleClickOutside.bind(instance),
+            delegateHandler,
```

Comment:
`bind` should only cause a problem inside the react class, so a better solution might be to have a top level
```
function delegateHandler(instance) {
  return instance.handleClickOutside.bind(instance);
}
```
and then here use:
```
... = ... = generateOutsideCheck(
  ReactDome.findDOMNode(instance),
  delegateHandler(instance),
  ...
);
```